src/components/navigation-header/components/dropdown-menu/index.tsx

```diff
@@ -204,105 +205,120 @@ const NavigationHeaderDropdownMenu = ({
 				id={menuId}
 				style={{ display: isOpen ? 'block' : 'none' }}
 			>
-				{itemGroups.map(
```

Comment:
Nit: Reducing the amount of inline return statements in this JSX could improve readability. In this case, `ItemGroup` or even `ItemGroups` could be defined as a separate component.